src/nixpacks/mod.rs

```diff
@@ -497,7 +497,7 @@ impl<'a> AppBuilder<'a> {
           FROM {base_image}
 
           RUN apt-get update && apt-get -y upgrade \\
-            && apt-get install --no-install-recommends -y locales curl xz-utils ca-certificates openssl \\
+            && apt-get install --no-install-recommends -y sudo locales curl xz-utils ca-certificates openssl \\
```

Comment:
What is the reasoning for installing `sudo` here? The apt packages below should be able to be installed without it

Reply:
`sudo` is needed to install the apt packages in the provider itself, as the setup phase runs as a separate build user instead of directly as `root`.